packages/web/components/EditProfileModal.tsx

```diff
@@ -91,67 +91,6 @@ const Label: React.FC<FormLabelProps> = React.forwardRef(
   },
 );
 
-const Input = React.forwardRef<typeof ChakraInput, InputProps>(
```

Comment:
You didn't like the inputs that resize themselves to the size of their contents?

They were capable of running off the side of the screen, so perhaps it's better if they leave…